service/routes.py

```diff
@@ -34,6 +34,27 @@ def index():
         status.HTTP_200_OK,
     )
     
+######################################################################
+# LIST ALL CUSTOMER
+######################################################################
+@app.route("/customers", methods=["GET"])
+def list_customers():
+    """Returns all of the customers"""
+    app.logger.info("Request for customer list")
+    customers = []
+    category = request.args.get("category")
+    name = request.args.get("name")
+    if category:
+        customers = Customer.find_by_category(category)
```

Comment:
I think category might be a property that's exclusive to pets, I don't think this field applies in this case